platform/macos/display_server_macos.mm

```diff
@@ -3607,18 +3607,20 @@
 			} else {
 				executable_command = vformat("open %s --args --rendering-driver opengl3", OS::get_singleton()->get_bundle_resource_dir().path_join("../..").simplify_path());
 			}
-			OS::get_singleton()->alert("Your video card driver does not support the selected Vulkan version.\n"
-									   "Please try updating your GPU driver or try using the OpenGL 3 driver.\n"
-									   "You can enable the OpenGL 3 driver by starting the engine from the\n"
-									   "command line with the command: '" +
-							executable_command + "'.\n"
-												 "If you have updated your graphics drivers recently, try rebooting.",
-					"Unable to initialize Video driver");
+			OS::get_singleton()->alert(
+					vformat("Your video card drivers seem not to support the required Vulkan version.\n\n"
+							"Please try updating your video card drivers or using the OpenGL 3 driver.\n\n"
```

Comment:
`update drivers`part probably should not be shown on macOS, since there are no standalone drivers, or maybe changed to `update macOS version`.

Reply:
Done.

Reply:
Also tweaked the iOS message for consistency (there doesn't seem to be an alert for Vulkan).